manifests/virt-manifest.yaml.in

```diff
@@ -0,0 +1,62 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: virt-manifest-service
+spec:
+  ports:
+    - port: 8186
+      targetPort: virt-manifest
+  externalIPs:
+    - "{{ master_ip }}"
+  selector:
+    app: virt-manifest
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: virt-manifest
+spec:
+  template:
+    metadata:
+      labels:
+        app: virt-manifest
+    spec:
+      containers:
+      - name: virt-manifest
+        image: {{ docker_prefix }}/virt-manifest:{{ docker_tag }}
+        imagePullPolicy: IfNotPresent
+        ports:
+          - containerPort: 8186
+            name: "virt-manifest"
+            protocol: "TCP"
+        volumeMounts:
+          - name: libvirt-data
+            mountPath: /var/lib/libvirt
+          - name: libvirt-runtime
+            mountPath: /var/run/libvirt
+        command:
+          - "/virt-manifest"
+          - "--port"
+          - "8186"
+      - name: manifest-libvirtd
+        image: {{ docker_prefix }}/libvirt-kubevirt:{{ docker_tag }}
+        imagePullPolicy: IfNotPresent
+        securityContext:
+          privileged: true
+          runAsUser: 0
+        env:
+        - name: LIBVIRTD_DEFAULT_NETWORK_DEVICE
+          value: eth1
+        volumeMounts:
+          - name: libvirt-data
+            mountPath: /var/lib/libvirt
+          - name: libvirt-runtime
+            mountPath: /var/run/libvirt
+        command: ["/libvirtd-limited.sh"]
+      volumes:
+      - name: libvirt-data
```

Comment:
Can't they be completely removed? What is the difference between mounting an empytDir vs not having that mount at all?